### ISSUE TITLE
FloatShield: The next one

### ISSUE THREAD

**Issue by gergelytakacs** [gergelytakacs/AutomationShield#44]
Please include all hardware design ideas and "bugs" that can be improved upon in the next version.

**Comment by gergelytakacs** [comment]
Smaller potentiometer (#19) a bit farther away from the fan for easier accesbility.

**Comment by kulho** [comment]
sensor and vent connectors too close together (possible solution - smaller vent connector size fits, for future PCBs might be redesigned as well)
short-circuited resistor (due to contacts touching usb port, has to be moved on the PCB in future design, for now, duct tape will do)

**Comment by gergelytakacs** [comment]
Notes from project presentation:
- Resistor over the USB connector, bad placement for shorts.
- Turbulent flow, use some sort of grid to make airflow more laminar. Grid available for download and print
- Use something else instead of the ball, the part could be custom designed for stability.
- Fan connector too close.
- Linking the Adafruit library.

**Comment by gergelytakacs** [comment]
@mgulan Just FYI, Peter Chmurciak a talented student from UAMM will take up work with FloatShield. He may do a thesis on it.

**Comment by mgulan** [comment]
@gergelytakacs Yep, I know him, that would be great.. unless he gets discouraged by JH.

**Comment by gergelytakacs** [comment]
@mgulan They already tried really hard:)

**Comment by mgulan** [comment]
@gergelytakacs :-)) and only one succeeded?

**Comment by gergelytakacs** [comment]
No, I mean, the "dark side" tried to discourage him, but so far the young Padavan has won the battles. ;)
